R/derive_param_computed.R

```diff
@@ -188,8 +203,23 @@
 #'   )
 #' )
 #'
-#' # Example 1b: Using option `keep_nas = TRUE` to derive MAP in the case where some/all
-#' # values of a variable used in the computation are missing
+#' @caption Example 1b - Adding a parameter computed from a formula with `keep_nas = TRUE`
```

Comment:
```suggestion
#' @caption Example 1b -  Using `keep_nas = TRUE`
```

I think this title would be more concise and would make the point of the subsection clearer

Reply:
Agreed! I wordsmithed this a bit - let me know what you think. Happy to use your suggestion verbatim instead if you'd prefer that!